kornia/augmentation/utils.py

```diff
@@ -0,0 +1,41 @@
+from typing import Tuple
+
+import torch
+
+
+def _transform_input(input: torch.Tensor) -> torch.Tensor:
+    r"""Reshape an input tensor to be (*, C, H, W). Accept either (H, W), (C, H, W) or (*, C, H, W).
+    Args:
+        input: torch.Tensor
+
+    Returns:
+        torch.Tensor
+    """
+    if not torch.is_tensor(input):
+        raise TypeError(f"Input type is not a torch.Tensor. Got {type(input)}")
+
+    if len(input.shape) not in [2, 3, 4]:
+        raise ValueError(
+            f"Input size must have a shape of either (H, W), (C, H, W) or (*, C, H, W). Got {input.shape}")
+
+    if len(input.shape) == 2:
+        input = input.unsqueeze(0)
+
+    if len(input.shape) == 3:
+        input = input.unsqueeze(0)
+
+    return input
+
+
+def _validate_input_shape(input: torch.Tensor, channel_index: int, number: int) -> bool:
```

Comment:
This is more a comment for future PR.  We had at some point the discussion about unifying checking functions, maybe this module could be the first start using such feature.